COPYING

```diff
@@ -5,23 +5,42 @@ Source License, version 1.0, or at your option, any later version ("BOSL"). See
 the file ./LICENSE-BOSL for the terms of the Bootstrap Open Source Licence,
 version 1.0.
 
+In this document,
+
+- "Zcash" means the blockchain defined by the most recent agreement between
+  the Electric Coin Company and the Zcash Foundation according to section 6.2 (b)
+  of the Zcash Trademark Agreement.
+- A "Zcash Chain Fork" means a blockchain that descends from the Zcash
+  blockchain and that activates its first change to the consensus rules 
+  relative to Zcash at a block height that is or was within 3456 blocks 
+  of the current height of the Zcash chain at the time of activation.
```

Comment:
The idea here is that the fork's implementation might have to hard-code information (e.g. a checkpoint or a snapshot of some of the chain state) that depends on the Zcash chain, and then have nodes running that code when the fork happens. This change allows 3 days between those two points, which is enough time to prepare the code, do a release, and have a reasonable number of nodes running.

Not all forks will need this; it depends how much they are changing. But it could plausibly be needed.